boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_common.dtsi

```diff
@@ -68,6 +70,12 @@
 	clock-frequency = <I2C_BITRATE_STANDARD>;
 	pinctrl-0 = <&i2c0_default>;
 	pinctrl-names = "default";
+
+	pfc8563_rtc: pfc8563@51 {
```

Comment:
I think this M5Stack board uses `bm8563` PN. If so, would you mind renaming `pfc8563` to `bm8563` instead? 

```c
	bm8563: bm8563@51 {
		compatible = "nxp,pcf8563";
		reg = <0x51>;
		status = "okay";
	};
```

Reply:
Oops, you're correct.
I referred to core2, but I'll fix it as follow the documentation.

Thank you for your review.